ts/components/conversation/SessionConversation.tsx

```diff
@@ -238,10 +277,34 @@ export class SessionConversation extends React.Component<Props, State> {
           <div className={classNames('conversation-info-panel', showMessageDetails && 'show')}>
             <MessageDetail />
           </div>
-
           {lightBoxOptions?.media && this.renderLightBox(lightBoxOptions)}
 
           <div className="conversation-messages">
+            {showMsgRequestUI && (
+              <ConversationRequestBanner>
+                <div className="conversation-request-banner__row">
```

Comment:
why don't you use a styled-component here too, instead of a normal div here and some css below at line 601?
```
.conversation-request-banner__row {
    display: flex;
    flex-direction: row;
    gap: var(--margins-lg);
    justify-content: center;
  }
```